app/views/TrusteeIndividualOrBusinessView.scala.html

```diff
@@ -34,11 +34,11 @@
 
         @components.error_summary(form.errors)
 
-        @components.heading("trusteeIndividualOrBusiness.heading")
+        @components.heading("trusteeIndividualOrBusiness.heading", Some(heading))
```

Comment:
Why do we have `Some(heading)` here? We don't in other instances of `@components.heading()`

Reply:
Fixed